### PR TITLE
Updated feedstock to version 0.6.1

### PR DESCRIPTION
**Pull request by theXYZT**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by theXYZT** [comment]
@conda-forge-admin, please rerender

**Comment by theXYZT** [comment]
@CurtLH Is there anything I am missing?

**Comment by theXYZT** [comment]
@conda-forge/staged-recipes Would like to merge this if possible but haven't received a review or feedback.

**Comment by theXYZT** [comment]
@conda-forge-admin, please ping team

**Comment by theXYZT** [comment]
@conda-forge-admin, please add user @theXYZT

**Review by zklaus — CHANGES_REQUESTED** [review]
We prefer to base builds on releases on Pypi rather than github tags. Since the new version is available on Pypi, could you please revert the changes in lines 1-10 that switch to the github tag? You can then also add your github username to the list of maintainers at the bottom.

**Comment by theXYZT** [comment]
@zklaus I can't figure out how to do that. On PyPi (https://pypi.org/project/nb2plots/#files), the only release file is a wheel. I can't find a link to a tarball.

**Comment by zklaus** [comment]
Sorry, @theXYZT, I think you are right. What an odd choice. I would suggest to raise the issue upstream so that source archives will be put on Pypi in the future, but you are right to move to the tags here, I guess.

I would suggest keeping the jinja2 variable (i.e. the old line 1) and hoping that upstream follows through in putting source releases out again.

**Comment by theXYZT** [comment]
@zklaus Done! Is this good enough to call this PR done?

File: recipe/meta.yaml
Comment (zklaus) [inline recipe/meta.yaml]:
```suggestion
  # As of 0.6.1, upstream puts only wheels on Pypi, hence we take the source directly from github.
  url: https://github.com/matthew-brett/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```

**Review by zklaus — APPROVED** [review]
I think it's ok. I left one small suggestion for the download and I still suggest that you add yourself to the list of maintainers here, i.e. simply add a line add the bottom with
```yaml
    - theXYZT
```

**Comment by theXYZT** [comment]
@zklaus Done! Hope that's good enough.

**Comment by zklaus** [comment]
@conda-forge/core, could somebody have a look here? The maintainer seems unresponsive (no reply after 3 months) and @theXYZT is willing to step up and help out.

**Comment by zklaus** [comment]
Thanks, @xhochy!

**Pull request merged**
